deepchecks/base/display_suite.py

```diff
@@ -46,11 +48,18 @@ def get_display_exists_icon(exists: bool):
     return '<div style="text-align: center">No</div>'
 
 
-def display_suite_result(suite_name: str, results: List[Union[CheckResult, CheckFailure]]):
+def display_suite_result(
+    suite_name: str,
+    results: List[Union[CheckResult, CheckFailure]],
+    output_order: Optional['suite.SuiteOutputOrder'] = None
```

Comment:
right now we have a single option order, so I don't think this is needed

Reply:
not sure if I understood you
in the ticket, you described two options?

how exactly it should be, `FAILED_FIRST` by default?